src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -2,9 +2,12 @@ package org.broadinstitute.dsde.firecloud.dataaccess
 
 import java.io.StringReader
 
-import com.google.api.client.googleapis.auth.oauth2.{GoogleAuthorizationCodeFlow, GoogleClientSecrets, GoogleTokenResponse}
```

Comment:
uh oh now you've seen how much I bastardized rawls' version of HttpGoogleServicesDAO!